components/Ui/DataTable/UiDataTableSection.vue

```diff
@@ -12,6 +22,7 @@
 interface Props {
   dataTableColumns: string[];
   sectionTitle: string;
+  sectionDescription: string;
```

Comment:
we want to make this an optional prop... which we do by adding a `?` behind the prop name

```suggestion
  sectionDescription?: string;
```

Reply:
resolved in https://github.com/Qiskit/qiskit.org/pull/3550/commits/9e7d698cbf2a0f898bd132f2575590cc22107a32